packages/legacy/core/App/screens/PINEnter.tsx

```diff
@@ -256,10 +256,32 @@ const PINEnter: React.FC<PINEnterProps> = ({ setAuthenticated, usage = PINEntryU
 
         if (!result) {
           const newAttempt = store.loginAttempt.loginAttempts + 1
-          if (!getLockoutPenalty(newAttempt)) {
+          const attemptsLeft = attemptLockoutThresholdRules.attemptIncrement - newAttempt 
+          if (!inlineMessages.enabled && !getLockoutPenalty(newAttempt)) {
             // skip displaying modals if we are going to lockout
             setAlertModalVisible(true)
           }
+          if (inlineMessages.enabled) {
```

Comment:
do we want this feature with the 'incorrect pin alert' as well? seems strange to keep the feature hidden unless inline messages are enabled

Reply:
I am wondering if we should remove the modal error messages and just use inline to simplify the code?

Otherwise agreed that the model error messages should display the same information @FarjadGov14

Reply:
sure, we okay with that @al-rosenthal ?